Semantic Scholar.js

```diff
@@ -37,148 +37,423 @@
+				"libraryCatalog": "Semantic Scholar",
+				"publicationTitle": "ArXiv",
+				"url": "https://www.semanticscholar.org/reader/204e3073870fae3d05bcbc2f6a8e263d9b72e776",
+				"attachments": [
+					{
+						"title": "Semantic Reader Link",
+						"mimeType": "text/html",
+						"snapshot": false
+					},
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [
+					"[TLDR] A new simple network architecture, the Transformer, based solely on attention mechanisms, dispensing with recurrence and convolutions entirely is proposed, which generalizes well to other tasks by applying it successfully to English constituency parsing both with large and limited training data."
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.semanticscholar.org/reader/fa72afa9b2cbc8f0d7b05d52548906610ffbb9c5",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Neural Machine Translation by Jointly Learning to Align and Translate",
+				"creators": [
+					{
+						"firstName": "Dzmitry",
+						"lastName": "Bahdanau",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Kyunghyun",
+						"lastName": "Cho",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Yoshua",
+						"lastName": "Bengio",
+						"creatorType": "author"
+					}
+				],
+				"date": "1 September 2014",
+				"abstractNote": "Neural machine translation is a recently proposed approach to machine translation. Unlike the traditional statistical machine translation, the neural machine translation aims at building a single neural network that can be jointly tuned to maximize the translation performance. The models proposed recently for neural machine translation often belong to a family of encoder-decoders and consists of an encoder that encodes a source sentence into a fixed-length vector from which a decoder generates a translation. In this paper, we conjecture that the use of a fixed-length vector is a bottleneck in improving the performance of this basic encoder-decoder architecture, and propose to extend this by allowing a model to automatically (soft-)search for parts of a source sentence that are relevant to predicting a target word, without having to form these parts as a hard segment explicitly. With this new approach, we achieve a translation performance comparable to the existing state-of-the-art phrase-based system on the task of English-to-French translation. Furthermore, qualitative analysis reveals that the (soft-)alignments found by the model agree well with our intuition.",
+				"libraryCatalog": "Semantic Scholar",
+				"publicationTitle": "CoRR",
+				"url": "https://www.semanticscholar.org/reader/fa72afa9b2cbc8f0d7b05d52548906610ffbb9c5",
```

Comment:
Would you be open to linking to both the Semantic Reader page and the Paper Detail Page (that's what we call the "catalog page")? We can do whatever you think is right. The reason we did it this way is because the request for better Zotero integration has specifically come from people using Semantic Reader, rather than people on the other pages, which we interpreted as users asking for easier back-and-forth navigation between Zotero and the Semantic Reader.

Just to confirm, if the user is saving to Zotero from the Paper Detail Page, we are currently only exporting the Paper Detail Page link. The Reader link is only exported if they save from the Reader.

Reply:
I'll leave the technical review to Abe, but just to contextualize, the `URL` field is what Zotero would use in a citation. That means
a) it should definitely be empty when there's no full text on semanticscholar and
b) I'd be inclined to say would seem odd to be for the semantic reader page. Otoh, I could see an attached link to the reader where users save from it useful?

Reply:
Got it, thanks for the context. Knowing that, it sounds like the Reader link should be an attachment if paper saved from the Reader, but the `URL` should be the Paper Detail Page link in all cases (just as you suggest).

Reply:
> but the URL should be the Paper Detail Page link in all cases (just as you suggest).

I'd say the Paper Detail Page should only be in the URL where SemanticScholar actually has the PDF. E.g. looking at something like
https://www.semanticscholar.org/paper/Multiple-Regression%3A-Testing-and-Interpreting-Toothaker/51cf7749204fdd6cddfc3fb1a1c7d94364656a47

that's really just a catalog entry and we shouldn't be saving a URL just as we're not saving a URL for a google scholar, PubMed, or library catalog entry.